packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -76,18 +81,21 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
   useLocalVideoStartTrigger(!!props.isVideoStreamOn);
   const VideoGalleryMemoized = useMemo(() => {
     return (
-      <VideoGallery
-        {...videoGalleryProps}
-        localVideoViewOptions={localVideoViewOptions}
-        remoteVideoViewOptions={remoteVideoViewOptions}
-        styles={VideoGalleryStyles}
-        layout="floatingLocalVideo"
-        showCameraSwitcherInLocalPreview={props.isMobile}
-        localVideoCameraCycleButtonProps={cameraSwitcherProps}
-        onRenderAvatar={onRenderAvatar}
-      />
+      <>
+        <Announcer announcementString={announcerString} ariaLive={'polite'} />
+        <VideoGallery
+          {...videoGalleryProps}
+          localVideoViewOptions={localVideoViewOptions}
+          remoteVideoViewOptions={remoteVideoViewOptions}
+          styles={VideoGalleryStyles}
+          layout="floatingLocalVideo"
+          showCameraSwitcherInLocalPreview={props.isMobile}
+          localVideoCameraCycleButtonProps={cameraSwitcherProps}
+          onRenderAvatar={onRenderAvatar}
+        />
+      </>
     );
-  }, [videoGalleryProps, props.isMobile, onRenderAvatar, cameraSwitcherProps]);
+  }, [videoGalleryProps, props.isMobile, onRenderAvatar, cameraSwitcherProps, announcerString]);
```

Comment:
From #2502 - I think we should avoid adding to the depedency array that rerenders video gallery as its our most expensive component

Reply:
Instead below instead of `return VideoGalleryMemoized;` you can 
return 
```
<> 
  <Announcer announcementString={announcerString} ariaLive={'polite'} />
  <VideoGalleryMemoized />
</>
```

Ideally we update the useMemo to a `React.memo` but that is a larger change we should keep out of this PR